onadata/apps/api/tasks.py

```diff
@@ -65,6 +68,14 @@ def delete_xform_async(xform_id):
     xform.soft_delete()
 
 
+def delete_user_async(username):
+    """Delete a user account task"""
+    user = User.objects.get(username=username)
+    # Delete user records from formbuilder app before deleting user object
+    fetch_user_formbuilder_details(user)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'user' in function call](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=4523955)